appholder/src/main/java/com/android/identity/wallet/transfer/TransferManager.kt

```diff
@@ -262,9 +264,7 @@ class TransferManager private constructor(private val context: Context) {
 
     fun sendResponse(deviceResponse: ByteArray, closeAfterSending: Boolean) {
         communication.sendResponse(deviceResponse, closeAfterSending)
-        if (closeAfterSending) {
-            disconnect()
-        }
+        disconnect(!closeAfterSending)
```

Comment:
This is wrong. The conditional call should still be there with a constant `false` as argument